packages/components/Tabs/src/Tabs.settings.ts

```diff
@@ -0,0 +1,26 @@
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { TabsType, tabsName } from './Tabs.types';
+
+export const settings: IComposeSettings<TabsType> = [
+  {
+    tokens: {
+      color: 'menuItemText',
+    },
+    root: {
+      accessible: true,
+      accessibilityRole: 'tablist',
+    },
+    label: {
+      style: {
+        fontSize: 16,
+        fontWeight: '600',
```

Comment:
Are these styles being pulled from somewhere?

Reply:
Talked about this. The styles are pulled in from WinUI figma files. I think we should make these more platform specific. So if you're adding a label for win32 maybe use `variant: BodySemibold`.

Reply:
For win32, it's now updated to `SubheaderSemibold`